src/miner.cpp

```diff
@@ -251,7 +252,7 @@ CBlockTemplate* Mining::CreateNewBlock(const CChainParams& chainparams) const
             if (!IsFinalTx(tx, nHeight, nLockTimeCutoff))
                 continue;
 
-            const uint64_t maxSigOps = Policy::blockSigOpAcceptLimit(nBlockSize + nTxSize);
+            const uint64_t maxSigOps = Policy::blockSigOpAcceptLimit(nBlockSize + nTxSize - nCoinbaseReserveSize);
```

Comment:
I am confused... nBlockSize == nCoinbaseReserveSize here... so this is essentially just Policy::blockSigOpAcceptLimit(nTxSize) which is not what I think you meant.